.gas-snapshot

```diff
@@ -20,34 +20,34 @@ Certificate_supportsInterface:test() (gas: 5159)
 Certificate_transferFrom:test() (gas: 48618)
 Certificate_transferFrom:test_reverts_when_paused() (gas: 38088)
 Certificate_transferFrom_reverts_ForbiddenTransferAfterMinting:test() (gas: 18960)
-Checkout_buyingFromOneRemoval:test() (gas: 727053)
-Checkout_buyingFromOneRemoval_byApproval:test() (gas: 685468)
-Checkout_buyingFromTenRemovals:test() (gas: 1976692)
+Checkout_buyingFromOneRemoval:test() (gas: 651442)
+Checkout_buyingFromOneRemoval_byApproval:test() (gas: 587334)
```

Comment:
some of these went down significantly because I took care of some todos to refactor setup behavior into the setup function